modules/lang-painless/src/main/java/org/elasticsearch/painless/lookup/PainlessLookupBuilder.java

```diff
@@ -374,6 +375,9 @@ public void addPainlessClass(Class<?> clazz, Map<Class<?>, Object> annotations)
                     }
 
                     canonicalClassNamesToClasses.put(importedCanonicalClassName.intern(), clazz);
+                    if (annotations.get(AliasAnnotation.class)instanceof AliasAnnotation alias) {
```

Comment:
nit: space before instanceof

Reply:
I prefer that as well, but spotless likes it this way.